framework/infrastructure/gcp/compute.py

```diff
@@ -189,6 +189,7 @@ def backend_service_patch_backends(
         backend_service,
         backends,
         max_rate_per_endpoint: Optional[int] = None,
+        circuit_breaker: Optional[Any] = None,
```

Comment:
Looks like it's a dict, can we at change `Any` to `Dict`?

Reply:
Done.